l10n_ir_base/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Iran - Base",
+    "version": "14.0.3.0.0",
+    "author": "saeed Raeisi, "
+    "Fawan",
+    "maintainer": "Saeed Raeisi",
+    "website": "https://fadoo.ir",
+    "license": "AGPL-3",
+    "category": "بومی سازی ایران/Technical",
+    "summary": "بومی سازی ایران ",
+    "depends": ["base","account"],
+    "data": [],
+        'external_dependencies': {
```

Comment:
extra identation

Reply:
I used the conversion library for this
If library use is not permitted
I can put the date conversion code in the module

Reply:
Using a library is fine, the code line has extra indentaton, and lint checs will fail.
By the way. make sure you install pre-commit in your local repo.
https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97